storageClient/storageClient.go

```diff
@@ -113,6 +127,60 @@ func (c *StorageClient) UpsertNoteByIDString(ctx context.Context, params Note) (
 	return result, err
 }
 
+// CreateGroupMembershipKey creates an encrypted version of the provided groupkey for the specified group member.
+func (c *StorageClient) CreateGroupMembershipKey(ctx context.Context, params CreateMembershipKeyRequest) (string, error) {
+	var wrappedEncryptedMembershipKey string
+	// Decrypt the access key
+	rawEncryptionKey, err := e3dbClients.DecodeSymmetricKey(c.EncryptionKeys.Private.Material)
+	if err != nil {
+		return wrappedEncryptedMembershipKey, err
+	}
+	groupKey, err := e3dbClients.DecryptEAK(params.EncryptedGroupKey, params.ShareePublicKey, rawEncryptionKey)
+	if err != nil {
+		return wrappedEncryptedMembershipKey, err
+	}
+	// Using the decrypted version of
+	// encrypted Group Key and the public key of
+	// the group admin specified in params,
+	// create an encrypted membership key that
+	// the new member can decrypt
+	wrappedEncryptedMembershipKey, err = c.WrapMembershipKeyForGroupMember(ctx, groupKey, params.NewMemberID)
+	return wrappedEncryptedMembershipKey, err
+}
+
+// WrapMembershipKeyForGroupMember tales the group key and wraps it for the new client
+func (c *StorageClient) WrapMembershipKeyForGroupMember(ctx context.Context, groupKey e3dbClients.SymmetricKey, newMemberID string) (string, error) {
```

Comment:
Do not change if this is inline with the naming for other keys!

In documentation, we typically refer to things like encyrpted access key. In the design I noted the name for this as 'encrypted membership key' to stay consistent. In most SDK these method names are things like `encryptMembershipKeyForGroupMember()`. I'd prefer to use 'encrypted' instead of 'wrapped'.

Noting again -- if the other methods for keys in clients go use 'wrapped' -- stick with that. I'd rather this code stay consistent internally.